tikv-client/src/main/java/com/pingcap/tikv/region/RegionSender.java

```diff
@@ -0,0 +1,97 @@
+/*
+ *
+ * Copyright 2017 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.pingcap.tikv.region;
+
+import com.pingcap.tikv.AbstractGRPCClient;
+import com.pingcap.tikv.TiSession;
+import com.pingcap.tikv.kvproto.Metapb.Store;
+import com.pingcap.tikv.kvproto.TikvGrpc;
+import com.pingcap.tikv.kvproto.TikvGrpc.TikvBlockingStub;
+import com.pingcap.tikv.kvproto.TikvGrpc.TikvStub;
+import io.grpc.ManagedChannel;
+
+public class RegionSender extends AbstractGRPCClient<TikvBlockingStub, TikvStub> implements RegionErrorReceiver  {
```

Comment:
What does this class exactly do? From its name, its a sender; but it does not actually have any send method.